src/jit/morph.cpp

```diff
@@ -9000,24 +8986,8 @@ GenTree* Compiler::fgMorphOneAsgBlockOp(GenTree* tree)
     bool       isCopyBlock    = asg->OperIsCopyBlkOp();
     bool       isInitBlock    = !isCopyBlock;
 
-    unsigned             size;
+    unsigned             size   = 0;
```

Comment:
Why was it changed? `size` was initialized on all paths before each use (we would have a compiler error if it was not).